docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM continuumio/anaconda3:4.1.1
+MAINTAINER "Greg Way" <gregway@mail.med.upenn.edu>
+
+# Install base packages and R
+RUN apt-get update --fix-missing && apt-get install -y \
+    libcurl4-openssl-dev \
+    libssl-dev \
+    libssh2-1-dev \
+    vim \
+    r-base \
+    r-base-dev \
+    r-cran-rcpp
+
+# Update to R 3.3.1
+RUN echo "deb http://cran.irsn.fr/bin/linux/debian jessie-cran3/" >> /etc/apt/sources.list
+RUN apt-key adv --keyserver keys.gnupg.net --recv-key 381BA480
+RUN apt-get update && apt-get install -y r-base
+
+# Clone TAD pathways GitHub repository
+RUN git clone https://github.com/greenelab/tad_pathways.git  
```

Comment:
> What is the goal in minting this image?

I'd say there are two goals:

1. To allow researchers access to the compute environment who want to build on the work
2. To allow anyone to reproduce the analysis

Since I will not be implementing continuous analysis in this repo (large file downloading costs) a docker image does not extend this need.

> You have a versioned anaconda environment, but then you have an unversioned clone URL?

I suppose this is an issue when python versions are updated, the build is not automatically rerun. I haven't looked into this, but do you know if its possible to automatically rebuild (--no-cache) and push a docker file if the environment changes? I can obviously do this manually, coinciding with at least with each release. Note that I actually don't have an `environment.yml` file yet (see #4). So I think this comment can be addressed simultaneously when I get to #4.